src/components/TextareaField/TextareaField.cy.tsx

```diff
@@ -0,0 +1,132 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Validation } from '@utilities/validation';
+import React from 'react';
+import { TextareaField } from './TextareaField';
+
+const DEFAULT_TEXT = 'A new text area';
+const PLACEHOLDER = 'This is the placeholder';
+const DECORATOR_TEXT = 'And also a decorator';
+const DECORATOR = <span>{DECORATOR_TEXT}</span>;
+const INPUT_TEXT = 'some text';
+const ROW_HEIGHT = 16;
+const TEXTAREA_ID = '[data-test-id=textarea]';
```

Comment:
I was thinking, should we expose and change the test id? This will lead to breaking changes but (hopefully) only on test files in `Clarify`/`web-app` project since devs shouldn't rely on `data-test-id` for the component features... WDYT?

Reply:
BTW, I exported the `data-test-id` prop but kept the default value `textarea` to avoid breaking changes